src/Discovery/Discover.cs

```diff
@@ -74,6 +74,60 @@ public bool BeginDiscovery()
             if (UserInputObj.CancellationContext.IsCancellationRequested)
                 UtilityFunctions.InitiateCancellation(UserInputObj);
 
+            int excelCreationPercentProgress = 0;
+            if (UserInputObj.AzureMigrateSourceAppliances.Contains("import"))
+                excelCreationPercentProgress = PerformImportDiscovery(masterSitesObj);
+            else
+                excelCreationPercentProgress = PerformApplianceDiscovery(masterSitesObj);

```

Comment:
For this to work properly,

please ensure that we are not sending any other appliance source with import, or add another condition that "import" is the only element in sources.